.github/workflows/deploy-staging.yml

```diff
@@ -13,15 +14,14 @@ jobs:
     uses: ./.github/workflows/linting.yml
     secrets: inherit
 
-  # deploy to staging
   scan-dev:
-    name: Zap scan of the dev site
+    name: Zap Scan
```

Comment:
Again, we probably want to explain the somewhat counterintuitive fact that we're scanning the dev environment during the staging deploy, and differentiate this step from the one below that has the identical name.